MIGRATING.md

```diff
@@ -0,0 +1,94 @@
+## Full list of classes that are no longer supported
+| Class                               | New Feature                                                                                                                                                    | Sample Codes directory/repository                                                 |
+|-------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------|
+| AIM (Authorize.NET/AIM)             | [PaymentTransactions](https://developer.authorize.net/api/reference/index.html#payment-transactions)                                                           | [sample-code-csharp/PaymentTransactions](https://github.com/AuthorizeNet/sample-code-csharp/tree/master/PaymentTransactions)    |
+| ARB (Authorize.NET/ARB)             | [RecurringBilling](https://developer.authorize.net/api/reference/index.html#recurring-billing)                                                                 | [sample-code-csharp/Recurring Billing](https://github.com/AuthorizeNet/sample-code-csharp/tree/master/RecurringBilling)          | 
+| CIM (Authorize.NET/CIM)             | [CustomerProfiles](https://developer.authorize.net/api/reference/index.html#customer-profiles)                                                                 | [sample-code-csharp/CustomerProfiles](https://github.com/AuthorizeNet/sample-code-csharp/tree/master/CustomerProfiles)          |
+| SIM (Authorize.NET/SIM)             | [Accept Hosted](https://developer.authorize.net/content/developer/en_us/api/reference/features/accept_hosted.html)                                             | Not available                                                                                                                         |
+| Reporting	(Authorize.NET/Reporting) | [TransactionReporting](https://developer.authorize.net/api/reference/index.html#transaction-reporting)                                                         | [sample-code-csharp/TransactionReporting](https://github.com/AuthorizeNet/sample-code-csharp/tree/master/TransactionReporting) |
+| CP (Authorize.NET/CP)             | [PaymentTransactions](https://developer.authorize.net/api/reference/index.html#payment-transactions)                                                           | [sample-code-csharp/PaymentTransactions](https://github.com/AuthorizeNet/sample-code-csharp/tree/master/PaymentTransactions)    |
+| DPM (Authorize.NET/DPM)             |[Accept.JS](https://developer.authorize.net/api/reference/features/acceptjs.html)                                                           | [Sample Accept Application](https://github.com/AuthorizeNet/accept-sample-app)    |
+
+## Example 
+#### Corresponding new model code (charge-credit-card):
```

Comment:
Change to --> #### Sample new model code for (charge-credit-card)

_No ending colon_

Reply:
### Summary
1. Change the heading for Corresponding new model code (marked in review) in MIGRATING.md.
2. Add an space after _inside_ in the file CONTRIBUTING.md

#### Lower priority
1. Remove extra new lines in the code in MIGRATING.md
2&3. Revert redundant changes in README.md